modules/nw-sriov-nic-mlx-secure-boot.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-nic-mlx-secure-boot_{context}"]
+= Configuring the SR-IOV Network Operator on Mellanox cards when Secure Boot is enabled
+
+The SR-IOV Network Operator supports an option to skip the firmware configuration for Mellanox (MLX) devices. It is currently the only way to create virtual functions by using the SR-IOV Network Operator, if the system has secure boot enabled. You are required to manually configure and allocate the number of virtual functions in the firmware before switching the system to secure boot.
+
+[NOTE]
+====
+The number of virtual functions in the firmware are the maximum number of virtual functions that you can request in the policy.
```

Comment:
```suggestion
The number of virtual functions in the firmware is the maximum number of virtual functions that you can request in the policy.
```

Reply:
Thanks @jeana-redhat for your feedback. I have applied your suggestion.